pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -695,6 +695,20 @@ func getServersConfig(cr *api.KieApp) ([]api.ServerTemplate, error) {
 			}
 
 			// Set replicas
+
+			if len(cr.Spec.Objects.Servers) > 0 {
```

Comment:
i believe this is given that servers is greater than 0, since we're already in a servers loop at this point.

Reply:
defensive mind (java ) here :)